src/Controller/UI/SideBySideController.php

```diff
@@ -0,0 +1,259 @@
+<?php
+
+namespace App\Controller\UI;
+
+use App\Handler\EntityHandler;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+use Symfony\Component\HttpKernel\Exception\HttpException;
+
+use App\Form\DatasetSubmissionType;
+
+use App\Entity\DatasetSubmission;
+use App\Entity\Dataset;
+
+/**
+ * The DIF controller for the Pelagos UI App Bundle.
+ */
+class SideBySideController extends AbstractController
+{
+
+    /**
+     * Valid values for $datasetFileTransferType and $metadataFileTransferType.
+     */
+    const SUBMISSIONS_STATES = array(
+        DatasetSubmission::STATUS_UNSUBMITTED => 'Unsubmitted',
+        DatasetSubmission::STATUS_INCOMPLETE => 'Draft',
+        DatasetSubmission::STATUS_COMPLETE => 'Submitted',
+        DatasetSubmission::STATUS_IN_REVIEW => 'In Review',
+    );
+
+    /**
+     * An entity handler instance.
+     *
+     * @var EntityHandler
+     */
+    protected $entityHandler;
+
+    /**
+     * SideBySideController constructor.
+     *
+     * @param EntityHandler $entityHandler An entity handler instance.
+     */
+    public function __construct(EntityHandler $entityHandler)
+    {
+        $this->entityHandler = $entityHandler;
+    }
+
+    /**
+     * The default action for Side by Side.
+     *
+     * @param Request     $request The Symfony request object.
+     * @param string|null $udi     The UDI of the Dataset to load.
+     *
+     * @Route("/sidebyside/{udi}", name="pelagos_app_ui_sidebyside_default")
```

Comment:
You need a method "GET" in this route, are else it will conflict with the other similar route below. (same url, but expects post). If you don't specify a method, it will accept all.

Reply:
We never specified the methods for GETs in the UI Controllers previously, Added GET methods to required UI Controllers